stories/profile/UserTestimonyListCard.stories.tsx

```diff
@@ -1,7 +1,106 @@
+import { useState } from "react"
+import { Form } from "react-bootstrap"
+import { Image } from 'components/bootstrap'
 import { createMeta } from "stories/utils"
+import styles from './UserTestimonyListCard.module.css'
 
 // TODO: move into components directory
-const UserTestimonyListCard = () => <div>TODO</div>
+const TestimonyCard = ({ testimony }) => {
+  const [showMore, setShowMore] = useState<boolean>(false)
+
+  const published = new Date(testimony.publishedAt * 1000).toLocaleDateString()
+  const positionTransform = {
+    endorse: {
+      color: "green",
+    },
+    oppose: {
+      transform: "scaleY(-1) rotate(-1turn)",
+    },
+    neutral: {
+      transform: "scaleY(-1) rotate(-0.5turn)",
+    },
+  }
+
+  const positionStyle = positionTransform[testimony.position]
+
+  return (
+    <div className="border-1 border-bottom border-dark px-3 py-2 d-flex flex-column">
+      <div className="d-flex align-items-center">
+        <span className="fs-2 pe-2">{testimony.billId}</span>
+        <Image
+          alt=""
+          src="thumbs-up.svg"
+          height={30}
+          style={positionStyle}
+        />
+      </div>
+      <div className="d-flex justify-content-between pb-3">
+        <span className={styles.testimony_card_title}>{testimony.title}</span>
+        <span><b>{published}</b></span>
+      </div>
+
+      {showMore ? (
+        <div className="pb-3">{testimony.content}</div>
+      ) : (
+        <>
+          <div className={styles.testimony_card_content}>{testimony.content.substring(0, 600)}</div>
+          <div className="py-2 align-self-end">
+            <span onClick={() => setShowMore(true)}>Show More</span>
+          </div>
+        </>
+      )}
+    </div>
+  )
+}
+
+const UserTestimonyListCard = () => {
```

Comment:
This component should receive all its display content through its props. Then in the story use [args](https://storybook.js.org/docs/react/writing-stories/args#story-args) to set the dummy data to display. In the app, we'll render this component with the actual testimony. The page-level component will use the hook to get the testimony, or we'll have a wrapper "connected" component with the hooks and stuff.